server/routes/deletePerson/deletePersonValidation.ts

```diff
@@ -1,15 +1,15 @@
 import { FormError } from '../../@types/template'
 
 const errors: { [key: string]: FormError } = {
-  MISSING_RADIO: {
+  DISCOVERY_MISSING_RADIO: {
```

Comment:
Should this error have been renamed? I'm unsure whether this ticket requires changes to the 'delete person' part of the app?

Reply:
reverted , well spotted !